### ISSUE TITLE
Add exception helper methods

### ISSUE THREAD

**Issue by KonH** [KonH/UnityRefAnalyzer#14]
It's annoying to write such code: https://github.com/KonH/UnityRefAnalyzer/blob/cfd657fa148584fd47729ccc2b0c7616511ddc63/RefAnalyzer/RefAnalyzer/Data/RefNode.cs#L20-L22
It's utility code, which required to don't spread errors in code deeply. And it need to be much more simple.
I suggest to use ExceptionHelper methods, which get's object, check it in some way (null, empty string) and raise exception from one place.

**Comment by wojciech-dabrowski** [comment]
I'll do it.

**Comment by KonH** [comment]
@wojciech-dabrowski why do you close your PR?

**Comment by KonH** [comment]
Nice work! But if you plan to do more PR, please use existing code-style. Now I fix it by myself and add .editorconfig to prevent misunderstanding and ease maintaining later.

**Comment by wojciech-dabrowski** [comment]
You are totally right. You could reject that or do some remarks in code review. Is there any things to do regarding this issue?
